README.md

```diff
@@ -1,38 +1,44 @@
-Statful Client for Vertx
+Statful Client for Vert.x
 ==============
 
 [![Build Status](https://travis-ci.org/statful/statful-client-vertx.svg?branch=master)](https://travis-ci.org/statful/statful-client-vertx)
 
-Statful client for Vertx. This client enables pushing custom metrics as well as metrics collection for [Vert.x](http://vertx.io/) based projects and report them to [Statful](http://statful.com/)
+Statful client for Vert.x written in Java. This client enables pushing custom metrics as well as metrics collection for [Vert.x](http://vertx.io/) based projects and sends them to [Statful.](http://statful.com/)
 
 This client leverages the capabilities provided by [Vert.x SPI](http://vertx.io/docs/apidocs/io/vertx/core/spi/metrics/VertxMetrics.html) to collect metrics.
 
-Please check the project pom.xml to see Vert.x version dependency since this will be a moving target, the aim is to always support the latest stable version.
+Please check the project’s [pom.xml](https://github.com/statful/statful-client-vertx/blob/master/pom.xml) file to see the current Vert.x version dependency since we aim always to support the latest stable version.
```

Comment:
should it be "since we aim to always support"

Reply:
Apparently splitting an infinitive verb is not very advisable, but as it is now I agree that it doesn't sound best. I'll change to "since we always aim to support..."